### DISCUSSION TITLE
how to see most recently modified files in the file list

### DISCUSSION THREAD

**Discussion by cedardocster** [Zettlr/Zettlr#4185] [General]
In the preferences I have both options checked off as "by modification date" and natural order, but still the top items in the list to the left start dating by 9 days ago and then 10 days ago, but there are lots of files in the folder younger than that.

is there a trick I'm missing?

I'm on version 2.3.0

**Comment by cedardocster** [comment]
also on Linux, I believe via the Deb

**Comment by nathanlesage** [comment]
You need to choose the sorting on a per-folder basis: right click on a folder, select properties, and then sort accordingly. The preferences options are only there to tell the app which property exactly to look at, and how to precisely sort depending on the option.

**Comment by cedardocster** [comment]
Even when I do that it still doesn't show right (will attach screenshots)

![Screenshot from 2023-03-03 11-20-30](https://user-images.githubusercontent.com/10538705/222798149-f07c0ea7-be11-4c9b-b952-af351f25ea84.png)
![Screenshot from 2023-03-03 11-21-03](https://user-images.githubusercontent.com/10538705/222798180-5513d633-2ce1-4b70-b21b-e0b9c71f7129.png)
![Screenshot from 2023-03-03 11-21-18](https://user-images.githubusercontent.com/10538705/222798207-ae1b4cc1-5bac-44be-8b19-61955ad9c395.png)
![Screenshot from 2023-03-03 11-21-43](https://user-images.githubusercontent.com/10538705/222798227-da9187a3-4858-4ad3-ac8b-11e6b4551924.png)

**Reply by nathanlesage** [reply]
Are you still on 2.3? I assume so, right? It could be that there was still some bug in this regard. In that case, I would direct you toward switching to the current 3.0 betas, in which that does not happen.

**Reply by cedardocster** [reply]
Thanks, I'll try that and get back to you

**Reply by cedardocster** [reply]
It works now! Thanks for the suggestion :)
